hack/generate-install-script.sh

```diff
@@ -35,4 +35,8 @@ write_block "sql/schema.sql"       sql/schema.sql
 echo "### TEMPLATES" >> $dst
 write_block "templates/hello-world-now.yaml" ./examples/templates/hello-world-now.yaml
 
+version=`git describe --tags $(git rev-list --tags --max-count=1)`
+sed -i.bak "s/DOCKER_TAG/$version/" ./$dst
+rm $dst.bak
```

Comment:
```suggestion
```

Reply:
I've added a comment in the code, for clarity

Reply:
The ```-i``` option is not POSIX compliant. So you can't expect it to work everywhere.

https://pubs.opengroup.org/onlinepubs/9699919799/utilities/sed.html

See #22